openedx/features/caliper_tracking/tests/current/edx.ui.lms.link_clicked.json

```diff
@@ -3,16 +3,16 @@
    "event_source":"browser",
    "name":"edx.ui.lms.link_clicked",
    "accept_language":"en-US,en;q=0.9,ur-PK;q=0.8,ur;q=0.7",
-   "time":"2018-10-16T14:23:24.785371+00:00",
+   "time":"2018-11-15T10:15:00.000Z",
```

Comment:
This is not the format that is present in the edx output. Please use correct input data.